internal/validator/auth/trust_root.go

```diff
@@ -19,6 +19,10 @@ type Keyless struct {
 	Issuer string `yaml:"issuer"`
 	// subject of the trust root (e.g. a mail address)
 	Subject string `yaml:"subject"`
+	// issuer regex of the trust root (e.g. an oidc provider)
+	IssuerRegex string `yaml:"issuerRegex"`
```

Comment:
I think we should validate that either issuer or issuesRegex is set, not both. Same for subject